compiler/ast/src/lib.rs

```diff
@@ -3,10 +3,7 @@ mod constant;
 #[cfg(feature = "fold")]
 mod fold_helpers;
 mod impls;
-#[cfg(feature = "unparse")]
-mod unparse;
```

Comment:
I think we used to depend on this, but maybe not anymore?

Reply:
Idk, it compiles :D